deployment/debian-package-x64/pkg-src/postinst

```diff
@@ -37,10 +38,15 @@ case "$1" in
     if [[ ! -d $LOGDATA ]]; then
       mkdir $LOGDATA
     fi
+    # ensure $CACHEDATA exists
+    if [[ ! -d $CACHEDATA ]]; then
+      mkdir $CACHEDATA
+    fi
     # Ensure permissions are correct on all config directories
     chown -R jellyfin:jellyfin $PROGRAMDATA
     chown -R jellyfin:jellyfin $CONFIGDATA
     chown -R jellyfin:jellyfin $LOGDATA
+    chown -R jellyfin:jellyfin $CACHEDATA
```

Comment:
Maybe condense things a little?
```suggestion
    chown -R jellyfin:jellyfin $PROGRAMDATA $CONFIGDATA $LOGDATA $CACHEDATA
```
Plus maybe `chmod o-rwx -R` to the same dirs? Privacy and stuff...

Reply:
Good idea, I'm confident that wouldn't have many bad implications.

Reply:
Done.